src/bot/eval.py

```diff
@@ -1,25 +1,30 @@
+"""
+This module contains functions
+for evaluating code using the Coliru API.
+"""
+
 import datetime
 import discord
 
 from discord.ext import commands
-from util import coliru
+from util import coliru  # pylint: disable=no-name-in-module
```

Comment:
If it's a local module (1) it should be `.util`, and (2) you can actually check that the value exists, and specifying it so that pylint recognizes it.